### PR TITLE
Newsletters: add 224 (2022-11-02)

### PR DESCRIPTION
**Pull request by harding**
- [x] @bitschmidty 
- [x] Bitcoin Core 23927 @Xekyo
- [x] Bitcoin Core 25957 @glozow
- [x] Other stuff @harding

### CODE REVIEW

File: _posts/en/newsletters/2022-11-02-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      if giving users more options is seen as better, there are many
+      aspects of relay policy that can be made configurable.  For
+      example, Bitcoin Knots provides a script pubkey reuse
+      (`spkreuse`) option for configuring a node to refuse to relay
+      any transactions which [reuse an address][topic output linking].
+
+    - *More permissive policies require widespread acceptance or better peering:*
+      A Bitcoin Core node makes eight outbound connections by default,
+      so at least 30% of the network needs to support a more
+      permissive policy before a node has 95% chance of finding at
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-11-02-newsletter.md:52]:
```suggestion
      permissive policy before a node has a 95% chance of finding at
```

File: _posts/en/newsletters/2022-11-02-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      least one randomly-selected peer that supports the same policy.
+      The fewer nodes that support a policy, the less likely it is
+      that a node will find a peer supporting that policy.
+
+    - *Better peering involves tradeoffs:* Bitcoin nodes can announce
+      their capabilities using the services field of the P2P `addr`,
+      [`addrv2`][topic addr v2], and `version` messages, allowing nodes
+      with common interests to find each other and form sub-networks
+      (called *preferential peering*).  Alternatively full node
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-11-02-newsletter.md:61]:
```suggestion
      (called *preferential peering*).  Alternatively, full node
```

File: _posts/en/newsletters/2022-11-02-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+      operators with common interests can use other software to form
+      independent relay networks (such as a network among LN nodes).
+      This can make relay effective even when just a few nodes
+      implement a policy, but nodes implementing a rare policy are
+      easier to identify and censor.  It also requires miners to be
+      join these sub-networks and alternative networks, raising the
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-11-02-newsletter.md:67]:
```suggestion
      easier to identify and censor.  It also requires miners to
      join these sub-networks and alternative networks, raising the
```

**Comment by renepickhardt** [comment]
I guess you may wish to cover https://twitter.com/brqgoo/status/1587397646125260802?t=ihdFNLORbmxCa-6NyFUFMQ&s=19 as a crucial and exploited bug in this weeks newsletter?

**Comment by glozow** [comment]
> I guess you may wish to cover https://twitter.com/brqgoo/status/1587397646125260802?t=ihdFNLORbmxCa-6NyFUFMQ&s=19 as a crucial and exploited bug in this weeks newsletter? (lnd nodes seem to be affected and their funds may be at risk because they don't sync with the chain anymore)

I imagine this would certainly be covered in next week's newsletter. Not to downplay the importance, but this is happening <24hr before publishing is scheduled and the newsletter usually covers things from the week before.

**Comment by renepickhardt** [comment]
I know that it was short notice but I decided to drop/suggest it here as I recall @harding once mentionig that for critical security upgrades the newsletter can have last minute hotfixes and updating lnd/btcd seems critical otherwise peers could steel funds.

**Comment by glozow** [comment]
Ah wasn't aware, ignore me heh

**Comment by LarryRuane** [comment]
> Ah wasn't aware, ignore me heh

But, just to be clear, we should _not_ ignore you when you say "ignore me" 😸

File: _posts/en/newsletters/2022-11-02-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23927][] restricts `getblockfrompeer` on pruned nodes
+  to heights below the node's current synchronization progress. This
+  prevents a footgun arising from retrieving future blocks making the
+  node's block-files ineligible for pruning.</br>
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-11-02-newsletter.md:170]: Not sure if the <br /> (slash after) is intended or if two newlines were intended here.

File: _posts/en/newsletters/2022-11-02-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+  Bitcoin Core stores blocks in files of about 130 MB in whatever order
+  it receives them. Pruning will discard entire block files, but will not
+  discard any file containing a block not processed by synchronization.
+  The combination of a small data allowance and repeated use of the
+  `getblockfrompeer` RPC could cause multiple block-files ineligible for
+  pruning, and cause a pruned node to exceed its data allowance.
+
+- [Bitcoin Core #25957][] improves the performance of rescans for
+  descriptor wallets by using the BIP157 block filter index (if enabled)
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-11-02-newsletter.md:179]: Perhaps 
```suggestion
  descriptor wallets by using the [BIP157][topic compact block filters] block filter index (if enabled)
```

**Comment by harding** [comment]
FYI, our publication policy is here: https://gist.github.com/harding/0dffdde00761262ca094219c33b8ff22  (yeah, I should probably move that into the repo at some point).  We'll add important security announcements at any time, e.g. we added [this](/en/newsletters/2018/09/18/#upgrade-to-bitcoin-core-0-16-3-to-fix-denial-of-service-vulnerability) less than an hour before publication.

In cases like the previous LND vulnerability where I also didn't learn about it until the day before publication, we announce the availability of a security critical fix immediately and then go into details in the subsequent week's newsletter.  I'm going to do the same for this week.

Thanks @renepickhardt for mentioning and everyone else for discussion!

**Comment by harding** [comment]
Edits made, thanks everyone!  Also added lede, updated releases/RCs with LND info, and added topic links.  Cheers!

**Comment by bitschmidty** [comment]
ACK https://github.com/bitcoinops/bitcoinops.github.io/pull/909/commits/8a50ac2b25d5714101a8539a6118d0b4a3533cfb sans a couple topic description fixups

**Comment by bitschmidty** [comment]
Another great newsletter thanks @harding @glozow @Xekyo for authoring and for @adamjonas @LarryRuane @glozow for reviewing! 🚀

**Pull request merged**